src/main/java/frc/robot/commands/ShootOne.java

```diff
@@ -104,18 +104,17 @@ public void initialize() {
 
   @Override
   public void execute() {
-    visionInfo = SmartDashboard.getNumberArray("vision/target_info", empty); // TODO: need actual vision info
-
 
     if (distance != 0.0) {
       foundTarget = true;
     }
 
     if (!foundTarget) {
-      distance = visionInfo[3];
+
+      distance = shooter.vision.getDistance();
     }
 
-    angleError = visionInfo[4] * 180 / 3.1416;
+    angleError = shooter.vision.getRobotAngle();
```

Comment:
Unrelated to this review, but we can still get here with distance==0. We probably should change the logic: 
if (!foundTarget) {
  check distance
}
if (distance !=0) {
  foundTarget = true;
  get angle error and do the rest
}